system_test/aest_sync_SUITE.erl

```diff
@@ -29,27 +29,38 @@
 
 %=== MACROS ====================================================================
 
--define(MINING_TIMEOUT, 1000).
+-define(MINING_TIMEOUT, 2000).
 
--define(NODE1, #{
-    name    => node1,
-    peers   => [node2],
+-define(OLD_NODE1, #{
+    name    => old_node1,
+    pubkey  => <<37,195,115,246,90,69,150,234,253,209,246,49,199,88,5,116,191,57,106,189,48,134,209,227,116,85,44,59,51,41,245,55>>,
+    peers   => [old_node2],
     backend => aest_docker,
     % Change to a compatible fixed version when possible
     source  => {pull, "aeternity/epoch:local"}
 }).
 
--define(NODE2, #{
-    name    => node2,
-    peers   => [node1],
+-define(OLD_NODE2, #{
+    name    => old_node2,
+    pubkey  => <<149,164,91,254,32,218,238,174,159,207,156,5,246,182,63,10,57,70,109,226,193,2,33,168,116,32,244,228,169,122,154,94>>,
+    peers   => [old_node1],
     backend => aest_docker,
     % Change to a compatible fixed version when possible
     source  => {pull, "aeternity/epoch:local"}
 }).
 
--define(NODE3, #{
-    name    => node3,
-    peers   => [node1],
+-define(NEW_NODE1, #{
+    name    => new_node1,
+    pubkey  => <<29,110,222,56,140,83,227,182,7,100,207,18,240,52,200,151,221,151,247,213,94,191,198,219,184,33,139,118,35,95,157,120>>,
+    peers   => [old_node1],
+    backend => aest_docker,
+    source  => {pull, "aeternity/epoch:local"}
+}).
+
+-define(STANDALONE_NODE, #{
+    name    => standalone_node,
+    pubkey  => <<29,110,222,56,140,83,227,182,7,100,207,18,240,52,200,151,221,151,247,213,94,191,198,219,184,33,139,118,35,95,157,120>>,
+    peers   => [],
```

Comment:
I ran successfully twice the system tests replacing all `:local` with `:v0.10.0`.
I get some distinct failures by replacing all `:loca` with `v0.10.1` - I suspect https://github.com/aeternity/epoch/commit/49eb621a51f669cbc33fe1f47d8d6872bfe7d9b4

Reply:
It looks like it does pick testnet because of no peers key. Maybe making peers as set as empty list helps. Last time I tried our yamerl lib (0.5.0) was not distinguishing between empty YAML sequence and empty YAML mapping, hence making using empty sequence not practical. Release 0.6.0 appears to have addressed that.

Reply:
> I get some distinct failures by replacing all :local with v0.10.1 - I suspect 49eb621

... that implies that system tests would fail in master if this PR 912 merged as is (though not hooked to CI yet).

I managed to get system tests to pass locally by:
* Tagging as `:local` v0.10.1 plus PR #942
* Amending system tests with following patch: 
```
diff --git a/system_test/aest_sync_SUITE_data/epoch.yaml.mustache b/system_test/aest_sync_SUITE_data/epoch.yaml.mustache
index e4d57536..76b66345 100644
--- a/system_test/aest_sync_SUITE_data/epoch.yaml.mustache
+++ b/system_test/aest_sync_SUITE_data/epoch.yaml.mustache
@@ -1,11 +1,9 @@
 {{#epoch_config}}
 ---
-{{#has_peers}}
-peers:
+peers:{{^peers}} []{{/peers}}
 {{#peers}}
     - {{peer}}
 {{/peers}}
-{{/has_peers}}

 sync:
     port: {{services.sync.port}}
diff --git a/system_test/helpers/aest_docker.erl b/system_test/helpers/aest_docker.erl
index a9512272..811a6b72 100644
--- a/system_test/helpers/aest_docker.erl
+++ b/system_test/helpers/aest_docker.erl
@@ -146,7 +146,6 @@ setup_node(Spec, BackendState) ->
     RootVars = #{
         hostname => Name,
         ext_addr => format("http://~s:~w/", [Hostname, ?EXT_HTTP_PORT]),
-        has_peers => length(PeerVars) > 0,
         peers => PeerVars,
         services => #{
             sync => #{port => ?EXT_SYNC_PORT},
```

Shall OLD_NODE1 and OLD_NODE1 be fixed to v0.10.1 for the v0.11.0 sprint? If yes, the patch above ^^^ is not sufficient.